Makefile

```diff
@@ -473,6 +473,8 @@ release-binary: $(RELEASE_DIR)
 .PHONY: release-staging
 release-staging: ## Builds and push container images to the staging bucket.
 	docker pull docker.io/docker/dockerfile:experimental
+	docker pull golang:1.13.12
```

Comment:
I'm wondering if we need to be explicit here, since the error was trying to pull `docker.io/library/golang:1.13.12`.

We can always patch it up if it complains, though.

Reply:
Seems to work locally, it might be defaulting